src/frontend/src/observer/observer_manager.rs

```diff
@@ -258,6 +258,13 @@ impl FrontendObserverNode {
                             ),
                             _ => panic!("receive an unsupported notify {:?}", resp),
                         },
+                        RelationInfo::Connection(connection) => match resp.operation() {
```

Comment:
Also need to re-initialize connections from snapshot as I mentions in `meta.proto`.